docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
+
+WORKDIR /go/src/github.com/hudl/fargo/
+RUN godep restore
+
+#UNcomment if you want to run tests on master branch.
+#CMD go test -v ./...
```

Comment:
Commented out as this checks out master branch of fargo. I just updated the Readme to say checkout the test branch then run tests. 

Regarding the IP addresses, yes unfortunately those IPs are hardcoded in tests, so I will let anyone familiar with fargo and tests fix that.